index.js

```diff
@@ -422,6 +438,12 @@ function TorrentGraph (root) {
     update()
   }
 
+  function on (event, cb) {
+    if (event === 'select') {
+      onSelect = cb
+    }
```

Comment:
The way this is implemented, only one event listener can be added. A reason to switch to the Node.js `EventEmitter` interface.